index.html

```diff
@@ -965,7 +965,7 @@ <h3>Queue a PressureObserver Task</h3>
         Set the <a href="https://tc39.es/ecma262/#surrounding-agent">surrounding agent</a>'s [=pressure observer task queued=] to true.
       </li>
       <li>
-        [=Queue=] a [=task=] on the [=PressureObserver task source=] to [[[#notify-observers]]].
+        [=Queue a global task=] on the [=PressureObserver task source=] to [[[#notify-observers]]].
```

Comment:
The "queue a global task" algorithm requires a global object in addition to a task source, so you need to specify one here (you already do it in `requestPermission()`, for example).